include/PaymentMethods.h

```diff
@@ -0,0 +1,56 @@
+
+// Created by Ahmed Hamdy Kotp
+
+#ifndef PAYMENTMETHODS_H
+#define PAYMENTMETHODS_H
+
+#include <string>
+#include <vector>
+#include "Date.h" 
+
+class PaymentMethods {
+
+private:
+    vector<Card> cards;
+    vector<DigitalWallet> wallets;
+    vector<BankTransfer> transfers;
```

Comment:
Separate each one in an independent class.
class Card, DigitalWallet, BankTransfer. All of them will inherit from PaymentMethod Class.
---
The user will write something like:
```C++
Card c{"1234-5678-9012-3456", 111, {1, 1, 2026}, 123};
c.showDetails();

// In User Class
vector<Card> cards;
vector<DigitalWallet> wallets;
vector<BankTransfer> transfers;
// User will have an array of each type of Payment Methods
```

Reply:
The PaymentMethod then will contain the general attributes and methods between all payment methods, such as currency type, transaction fees, time constraints between transactions, and security factors.

Reply:
Yes, this idea looks pretty good,  but my back will be brocken unfortunately 😭

Reply:
Why my friend? it's just some constructors, getters, setters, and methods 😅😅. Don't get tired now, we still have a lot of work !!🔥🔥

Reply:
You already made almost 75% of the work.